src/module/postgame.js

```diff
@@ -0,0 +1,62 @@
+const name = document.getElementById('name');
+const score = document.getElementById('score');
+const submit = document.getElementById('submit');
+const ul = document.getElementById('ul');
+const refresh = document.getElementById('refresh');
+
+export const getGameId = localStorage.getItem('gameId');
+
+export const postData = (gameId) => {
+  const url = `https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${gameId}/scores`;
+  return fetch(url, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({
+      user: name.value,
+      score: score.value,
+    }),
+  })
+    .then((response) => response.json())
+    .catch((error) => {
+      throw error;
+    });
```

Comment:
According to the requirements, you are expected to use async and await in all requests as they are much cleaner and improve the readability of your code.

NOTE: Please make this adjustments in every other places where requests are being made